### PR TITLE
New fast-integration test skr-without-sc-integration-dev

### PR DESCRIPTION
**Pull request by wozniakjan**
**Description**
This is a duplicate of the original `skr-test`, with only difference being the test is stripped of any references to service catalog. Other tests using `skr-test` will eventually be ported to referencing this `skr-without-sc-integration-dev`.

logs from pj tester execution: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/logs/wozniakjan_test_of_prowjob_skr-aws-without-sc-integration-dev/1518940992261066752

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/13415

**Depends on**
- [x] https://github.com/kyma-project/kyma/pull/13801

### CODE REVIEW

File: templates/data/skr-without-sc-integration-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          keb_plan_aws:
+            annotations:
+              pipeline.platform: gardener_aws
+            env:
+              KEB_PLAN_ID: "361c511f-f939-4621-b228-d0fb79a1fe15"
+        jobConfigs:
+          - repoName: "kyma-project/kyma"
+            jobs:
+              - jobConfig:
+                  name: "skr-aws-integration-dev"
```
Comment (tillknuesting) [inline templates/data/skr-without-sc-integration-dev.yaml:36]: change to"skr-aws-without-sc-integration-dev"

**Review by tillknuesting — APPROVED** [review]

**Comment by dzahariev** [comment]
@tillknuesting, the "vpath/pjtester.yaml" need to be removed from PR before merge to be allowed!

**Review by tillknuesting — APPROVED** [review]

**Review by dekiel — APPROVED** [review]

**Pull request merged**